backend/db/seed.sql

```diff
@@ -281,16 +281,16 @@ INSERT INTO volunteers
-    ('Bonnie', 'Simpson', 'bsimpson', 'bsimpson@gmail.com', 'https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/1589506898600-Bonnie%20Simpson.png', 'The New York Times', 'thenewyorktimes', 'CEO', FALSE, FALSE, FALSE, TRUE, TRUE, FALSE, TRUE, TRUE, TRUE, 'https://www.linkedin.com/in/'),
-    ('Joseph', 'Parker', 'jparker', 'jparker@gmail.com', 'https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/1589505381205-Joseph%20Parker.jpg', 'Spotify', 'spotify', 'Web Designer', TRUE, TRUE, FALSE, TRUE, TRUE, FALSE, TRUE, FALSE, FALSE, 'https://www.linkedin.com/in/'),
-    ('Kenneth', 'Wood', 'kwood', 'kwood@gmail.com', 'https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/1589505205590-Kenneth%20Wood.png', 'Google', 'google', 'HR', FALSE, FALSE, TRUE, FALSE, TRUE, FALSE, TRUE, TRUE, FALSE, 'https://www.linkedin.com/in/'),
-    ('Thomas', 'Potter', 'tpotter', 'tpotter@gmail.com', 'https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/1589500889403-Thomas%20Potter.png', 'Google', 'google', 'Senior Software Engineer', TRUE, TRUE, FALSE, FALSE, TRUE, FALSE, FALSE, FALSE, TRUE, 'https://www.linkedin.com/in/');   -- 10
+    ('Daniel', 'Lopez', 'dlopez', 'dlopez@gmail.com', 'https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/DanielLopez.jpg', 'Capital One', 'capitalone', 'Software Engineer', TRUE, TRUE, FALSE, FALSE, TRUE, FALSE, FALSE, TRUE, TRUE, 'https://www.linkedin.com/in/daniellopez/'),
+    ('Christina', 'Hall', 'chall', 'chall@gmail.com', 'https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/ChristinaHall.jpg', 'Capital One', 'capitalone', 'Mobile Software Engineer', TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, 'https://www.linkedin.com/in/'),
+    ('Judith', 'Simmons', 'jsimmon', 'jsimmons@gmail.com', 'https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/JudithSimmons.png', 'WayFair', 'wayfair', 'Software Engineer', TRUE, TRUE, TRUE, FALSE, TRUE, FALSE, FALSE, TRUE, TRUE, 'https://www.linkedin.com/in/'),
+    ('Catherine', 'Barnes', 'cbarnes', 'cbarnes@gmail.com', 'https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/CatherineBarnes.jpg', 'JustWorks', 'justworks', 'Software Engineer', FALSE, TRUE, TRUE, TRUE, FALSE, FALSE, TRUE, FALSE, TRUE, 'https://www.linkedin.com/in/'),
+    ('Pamela', 'Young', 'pyoung', 'pyoung@gmail.com', 'https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/PamelaYoung.png', 'JPMorganChase', 'jpmorganchase', 'Hiring Manager', FALSE, FALSE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, TRUE, 'https://www.linkedin.com/in/'),    -- 5
+    ('John', 'Evans', 'jevans', 'jevans@gmail.com', 'https://pursuit-volunteer-management.s3.us-east-2.amazonaws.com/JohnEvans.png', 'Google', 'google', 'Tech Lead', TRUE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, TRUE, 'https://www.linkedin.com/in/'),
```

Comment:
John Evans's profile picture doesn't seem to work for me. Please double check file and filename of it. Thanks!

Reply:
The link on the seed seems right I’ll take a look at the bucket later this evening maybe I didn’t name it right there or could me I lost the picture by mistake
Just in case @Briahana-Mauge could you please slack me the picture file for John Evan?